### ISSUE TITLE
Previous issues and menu items cannot be accessed by voiceover

### ISSUE THREAD

**Issue by avaseghi** [wenqili/adjacent#14]
Items in previous issue and menu are not displayed and unable to be read by voiceover.

**Comment by wenqili** [comment]
- [x] Refactored Navbar, added toggle to Navbar on mobile and desktop
- [x] added keyboard navigation
- [x] added skip navigation
